service/service.go

```diff
@@ -108,19 +153,21 @@ func New(config Config) (*Service, error) {
 
 	newService := &Service{
 		// Dependencies.
-		Deployer: deployerService,
-		Version:  versionService,
+		Deployer:   deployerService,
+		HelmClient: helmClient,
+		Version:    versionService,
 	}
 
 	return newService, nil
 }
 
-type Service struct {
-	// Dependencies.
-	Deployer deployer.Deployer
-	Version  *version.Service
-}
+func (s *Service) Boot(ctx context.Context) error {
```

Comment:
This is the actual change to call `EnsureTillerInstalled`. @kopiczko since this is called from a goroutine I think its best just to panic if there is an error. WDYT?

Reply:
Yes I think it makes sense for now. But please create a TODO issue (and a TODO comment here) saying that we shouldn't panic there and ping me in the issue. I have some ideas in my head but they need further discussion and there is no point in blocking your progress now.